packages/authorization-process/src/services/validators.ts

```diff
@@ -31,29 +31,46 @@ export const assertUserSelfcareSecurityPrivileges = async ({
   selfcareV2InstitutionClient,
   userIdToCheck,
   correlationId,
+  roles,
 }: {
   selfcareId: string;
   requesterUserId: UserId;
   consumerId: TenantId;
   selfcareV2InstitutionClient: SelfcareV2InstitutionClient;
   userIdToCheck: UserId;
   correlationId: CorrelationId;
+  roles: UserRole[];
 }): Promise<void> => {
-  const users =
-    await selfcareV2InstitutionClient.getInstitutionProductUsersUsingGET({
-      params: { institutionId: selfcareId },
-      queries: {
-        userIdForAuth: requesterUserId,
-        userId: userIdToCheck,
-        productRoles: [userRoles.SECURITY_ROLE, userRoles.ADMIN_ROLE],
-      },
-      headers: {
-        "X-Correlation-Id": correlationId,
-      },
-    });
-  if (users.length === 0) {
+  // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+  const role = () => {
+    if (roles.includes(userRoles.ADMIN_ROLE)) {
+      return userRoles.ADMIN_ROLE;
+    }
+    if (roles.includes(userRoles.SECURITY_ROLE)) {
+      return userRoles.SECURITY_ROLE;
+    }
     throw userWithoutSecurityPrivileges(consumerId, requesterUserId);
-  }
+  };
+
+  await selfcareV2InstitutionClient.getInstitutionUsersByProductUsingGET({
+    params: { institutionId: selfcareId },
+    queries: {
+      userId: userIdToCheck,
+      productRoles: role(), // At this point productRoles is a string and not a string[], which is why I preferred to bring the value out
```

Comment:
`productRoles` is a string now, but how do we pass more than one productRole? So we need to serialize it "comma-separated"? Idk @beetlecrunch 
We went in the [selfcare impl](https://github.com/pagopa/selfcare-external-api-backend/blob/main/src/main/java/it/pagopa/selfcare/external_api/controller/InstitutionV2Controller.java#L115), and the `productRoles` is a `Set<string>`

Reply:
I confirm that the query params is now a `comma-separated` string value.
Example `api, security,admin`